File: help/eng_gb/exercicios_da_ficha_5/Ficha05_008/Ficha05_008.js
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

//Main Function
function main() {
    document.write("Minimum : " );// print text
    imin = parseInt(prompt("Minimum : " ,"0") );
    
    document.write(imin + "\n");// print input 
    document.write("Maximum : " );// print text
    imax = parseInt(prompt("Maximum : " ,"0") );
    
    document.write(imax + "\n");// print input 
    for( var i = imin; i < imax + 1; i = i + 1 ) {
        if(i % 2 == 0 ){
            document.write(i + " " );
        }
    }
}

//JavaScript  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Sat, 2022 Sep 24 22:05

File: help/eng_gb/exercicios_da_ficha_5/Ficha05_008/Ficha05_008.cs
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

using System.IO;
using System;
public class Ficha05_008 {

    //Main Function
    static void Main() {
        Console.Write("Minimum : " );
        int imin = int.Parse(Console.ReadLine());
        Console.Write("Maximum : " );
        int imax = int.Parse(Console.ReadLine());
        for( int i = imin; i < imax + 1; i = i + 1 ) {
            if(i % 2 == 0 ){
                Console.Write(i + " " );
            }
        }
    }

}
//C#  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Sat, 2022 Sep 24 22:05
